pom.xml

```diff
@@ -627,4 +628,11 @@
             <url>https://repo.b3p.nl/nexus/repository/public/</url>
         </repository>
     </repositories>
+    <licenses>
+        <license>
+            <name>MIT</name>
+            <distribution>repo</distribution>
+            <url>https://spdx.org/licenses/MIT.html</url>
```

Comment:
dit moet de daadwerkelijke license file voor het project zijn in plain text omdat die wordt opgenomen in de maven site. Nu wijst deze naar een template

Reply:
Oh dat wist ik niet, ik zag bij de documentatie hierover alleen dat het een URL was. Deze URL kan ook naar het LICENSES/MIT.txt wijzen.